### ISSUE TITLE
Custom Sounds and Apps no longer working in Administration

### ISSUE THREAD

**Issue by bkraul** [RocketChat/Rocket.Chat#12554]
### Description:
The admin sections to customize Custom Sounds and Apps no longer work as of the latest develop update.

### Steps to reproduce:

1. Update develop to latest (0.72-develop. commit: b4debdc55d599cf3a9c2a31be1f89b9972bea5b5)
2. Go to Administration
3. Click on Custom Sounds, or Apps.

### Expected behavior:

The sections for custom sounds or apps should appear.

### Actual behavior:

Nothing happens.

### Demo Video
[![Video](https://i.ytimg.com/vi/zI2EL86ZJw8/hqdefault.jpg)](https://youtu.be/zI2EL86ZJw8 "Video")

### Rocket.Chat

| Version                 | 0.72.0-develop                                            |
| ----------------------- | --------------------------------------------------------- |
| Database Migration      | 135                                                       |
| Database Migration Date |                                                           |
| Installed at            | Wed Nov 07 2018 07:08:18 GMT-0600 (Central Standard Time) |
| Uptime                  | 17 seconds                                                |
| Deployment ID           | ZMnphP2xrSgdJk4FK                                         |
| PID                     | 1                                                         |
| Running Instances       | 1                                                         |
| OpLog                   | Enabled                                                   |

### Commit

| Hash    | b4debdc55d599cf3a9c2a31be1f89b9972bea5b5                     |
| ------- | ------------------------------------------------------------ |
| Date    | Tue Nov 6 14:56:18 2018 -0200                                |
| Branch  | develop                                                      |
| Tag     | 0.71.0                                                       |
| Author  | Marcos Spessatto Defendi                                     |
| Subject | Convert rocketchat-autotranslate to main module structure (#12530) |

### Runtime Environment

| OS Type         | Linux                 |
| --------------- | --------------------- |
| OS Platform     | linux                 |
| OS Arch         | x64                   |
| OS Release      | 4.15.0-38-generic     |
| Node Version    | v8.11.3               |
| OS Uptime       | 4 minutes, 18 seconds |
| OS Load Average | 0.54, 0.29, 0.13      |
| OS Total Memory | 3.85 GB               |
| OS Free Memory  | 1.69 GB               |
| OS CPU Count    | 2                     |

### Additional context

This happened on a brand new docker installation, and also on an existing installation that had the sections previously working (it works on commit 265669d9805722912412e02ca4171f12f514be94)
, and do not work now because of the update.

### Relevant logs:

Logs show no errors.

**Comment by tassoevan** [comment]
Both are logging the following error on browser console:

```
Error: No such template: requiresPermission
    at http://localhost:3000/packages/blaze.js?hash=adc5286b78e5c0f8e7f56a602f77eefb5def6bf1:3212:15
    at Blaze.View.<anonymous> (http://localhost:3000/packages/spacebars.js?hash=3fe9af696e128faeb64cb3b88e59dcf3d4ced009:68:23)
    at http://localhost:3000/packages/blaze.js?hash=adc5286b78e5c0f8e7f56a602f77eefb5def6bf1:1934:20
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?hash=adc5286b78e5c0f8e7f56a602f77eefb5def6bf1:3769:14)
    at http://localhost:3000/packages/blaze.js?hash=adc5286b78e5c0f8e7f56a602f77eefb5def6bf1:1932:29
    at Object.Blaze._withCurrentView (http://localhost:3000/packages/blaze.js?hash=adc5286b78e5c0f8e7f56a602f77eefb5def6bf1:2271:12)
    at viewAutorun (http://localhost:3000/packages/blaze.js?hash=adc5286b78e5c0f8e7f56a602f77eefb5def6bf1:1931:18)
    at Computation._compute (http://localhost:3000/packages/tracker.js?hash=725573cf35451add9ec258f882b90fd0455ebc45:332:38)
    at new Computation (http://localhost:3000/packages/tracker.js?hash=725573cf35451add9ec258f882b90fd0455ebc45:229:12)
    at Object.Tracker.autorun (http://localhost:3000/packages/tracker.js?hash=725573cf35451add9ec258f882b90fd0455ebc45:602:11) undefined
```

**Comment by bkraul** [comment]
This appears to have been fixed on commit 6db43602e94211e84c503e92eb0a4040be0d92ef . Thanks guys!
